### PR TITLE
Improvement/documentation

### PR DESCRIPTION
**Pull request by MALPI**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 ## Purpose
 
-This project provides a small cassandra client.
+This project provides a lightweight auto-configuration of the Datastax cassandra client for Spring-Boot projects.
+
+It's using the latest version of the [Datastax Java Driver](http://docs.datastax.com/en/developer/java-driver/3.0/common/drivers/reference/driverReference_r.html).
```
Comment (danielrohe) [inline README.md]: afaik does spring-boot uses an older version in the properties section of the pom <cassandra-driver.version>2.1.9</cassandra-driver.version>, can you check by adjusting the property whether it runs with version 3.0.x?

Comment (danielrohe) [inline README.md]: I've tested it and Datastax renamed AddressTranslater (2.1.9) to AddressTranslator (3.0.0) which breaks compilation.

Comment (MALPI) [inline README.md]: Then we should consider Injecting this property on another way. We currently implement against version 3.0.0 of datastax.

Comment (MALPI) [inline README.md]: Besides that I don't think that this property is really useful for us, compared with the benefit of having a lib which can be used with the latest version of datastax.

From the [documentation] (http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/policies/AddressTranslator.html):

` For each node, the address the driver will receive will correspond to the address set as rpc_address in the node yaml file. In most case, this is the correct address to use by the driver and that is what is used by default.`

I don't really think that this needs to be configurable?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,12 @@
 
 ## Purpose
 
-This project provides a small cassandra client.
+This project provides a lightweight auto-configuration of the Datastax cassandra client for Spring-Boot projects.
```
Comment (bocytko) [inline README.md]: You should also explicitly outline the differences compared to spring's [CassandraAutoConfiguration](http://docs.spring.io/spring-boot/docs/1.3.2.RELEASE/api/org/springframework/boot/autoconfigure/cassandra/CassandraAutoConfiguration.html).

File: README.md [hunk 3]
```diff
@@ -2,7 +2,12 @@
...
+
+It's using the latest version of the [Datastax Java Driver](http://docs.datastax.com/en/developer/java-driver/3.0/common/drivers/reference/driverReference_r.html).
+
+It offers the possibility to configure the contact points of cassandra directly in the application `application.properties` or to define a key which will be looked up
+in the configured ETCD-Cluster.
```
Comment (bocytko) [inline README.md]: formatting: etcd-cluster?

File: README.md [hunk 4]
```diff
@@ -2,7 +2,12 @@
 
 ## Purpose
 
-This project provides a small cassandra client.
+This project provides a lightweight auto-configuration of the Datastax cassandra client for Spring-Boot projects.
+
+It's using the latest version of the [Datastax Java Driver](http://docs.datastax.com/en/developer/java-driver/3.0/common/drivers/reference/driverReference_r.html).
+
+It offers the possibility to configure the contact points of cassandra directly in the application `application.properties` or to define a key which will be looked up
```
Comment (bocytko) [inline README.md]: `s/cassandra/Cassandra/g` ?

**Pull request closed without merging**